docs/csharp/misc/cs1107.md

```diff
@@ -17,20 +17,22 @@ ms.author: "wiwagn"
 # Compiler Error CS1107
 A parameter can only have one 'modifier name' modifier.  
   
- It is an error for parameter modifiers such as `this`, `ref`, and `out` to appear more than one time in a parameter definition.  
+ It is an error for parameter modifiers such as `this`, `ref`, `in`, and `out` to appear more than one time in a parameter definition.  
   
 ## Example  
  The following example generates CS1107:  
   
 ```  
 // cs1107.cs  
-public static class Test  
+public class Test  
```

Comment:
A type that contains extension methods has to be `static`, no?

Reply:
Good catch.